app/database_etl/owid_ingestion_handler/owid_data_handler.py

```diff
@@ -93,6 +98,35 @@ def get_vaccinations_per_hundred(country_name: str, midpoint_date: datetime, ful
     return float(ret) if not ret.empty else None
 
 
+def get_vaccination_policy(country_name: str, midpoint_date: datetime):
+    if str(midpoint_date) == 'NaT': return None
+    country_id = get_country_code(country_name)
+    offset = OFFSETS['VACCINATIONS']
+    target_date = (midpoint_date + offset).strftime('%Y%m%d')
+
+    country_data = vaccine_policy_rollout_df.loc[vaccine_policy_rollout_df['CountryCode'] == country_id]
+    country_data['Date'] = country_data['Date'].apply(lambda x: str(x))
+    ret = country_data.loc[(country_data['Date'] == target_date) &
+                           (country_data['Jurisdiction'] == 'NAT_TOTAL')]['H7_Vaccination policy']
+
+    if ret.empty or pd.isna(ret.iloc[0]):
+        return None
+
+    # Map the ordinal value of the vaccination policy to the policy level
+    vaccination_policy_mapping = {0: "No availability",
```

Comment:
This would be worth defining outside the function.

Reply:
do you think we will use it elsewhere? if not, think we can leave it in the function

Reply:
Prob not used elsewhere. Thought just being that if you define it in the function, won't it get re-defined every time the function runs? Could be wrong